x86_64-w64-mingw32,arm-cortexa9_neon-linux-gnueabihf build failure

Trying to be a bit more pro-active about running build-all rather than waiting for a release.

The  x86_64-w64-mingw32,arm-cortexa9_neon-linux-gnueabihf config fails on master (currently 1e9bf815) , it worked on 1.27.0. Probably one of the recent updates has affected it.

```
[ALL  ]    /home/ctng/x-tools/x86_64-w64-mingw32/lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld: /home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/buildtools/complibs-host/lib/libncursesw.a(lib_termcap.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/ncurses/ncurses/tinfo/lib_termcap.c:54: multiple definition of `UP'; ../readline/readline/libreadline.a(terminal.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/gdb/readline/readline/terminal.c:109: first defined here
[ALL  ]    /home/ctng/x-tools/x86_64-w64-mingw32/lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld: /home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/buildtools/complibs-host/lib/libncursesw.a(lib_termcap.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/ncurses/ncurses/tinfo/lib_termcap.c:55: multiple definition of `BC'; ../readline/readline/libreadline.a(terminal.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/gdb/readline/readline/terminal.c:109: first defined here
[ALL  ]    /home/ctng/x-tools/x86_64-w64-mingw32/lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld: /home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/buildtools/complibs-host/lib/libncursesw.a(lib_tputs.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/ncurses/ncurses/tinfo/lib_tputs.c:57: multiple definition of `PC'; ../readline/readline/libreadline.a(terminal.o):/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/src/gdb/readline/readline/terminal.c:109: first defined here
[ERROR]    collect2: error: ld returned 1 exit status
[ERROR]    make[4]: *** [Makefile:2232: gdb.exe] Error 1
[ALL  ]    make[4]: Leaving directory '/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/build/build-gdb-cross/gdb'
[ERROR]    make[3]: *** [Makefile:11036: all-gdb] Error 2
[ALL  ]    make[3]: Leaving directory '/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/build/build-gdb-cross'
[ERROR]    make[2]: *** [Makefile:1042: all] Error 2
[ALL  ]    make[2]: Leaving directory '/home/ctng/crosstool-ng/.build/HOST-x86_64-w64-mingw32/arm-cortexa9_neon-linux-gnueabihf/build/build-gdb-cross'
```

[build.log.gz](https://github.com/user-attachments/files/18804994/build.log.gz)

--- a/scripts/build/companion_libs/220-ncurses.sh
+++ b/scripts/build/companion_libs/220-ncurses.sh
@@ -133,6 +133,8 @@ do_ncurses_backend() {
         esac
     done
 
+    ncurses_opts+=("--disable-widec")
+
     if [ "${CT_NCURSES_NEW_ABI}" != "y" ]; then
         ncurses_opts+=("--with-abi-version=5")
     fi

